src/test/librados/crimson_env.h

```diff
@@ -0,0 +1,12 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include <cstdlib>
+
+#define CHECK_ENV();                    \
+  if (getenv("CRIMSON_COMPAT")) {       \
+    return;                             \
```

Comment:
I don't think returning silently is a good. We should be able to differentiate between _green_ and _skipped_ tests. IIUC now everything would be green.

How about [`GTEST_SKIP()`](http://google.github.io/googletest/advanced.html#skipping-test-execution)?

Anyway, it's just a detail. I like the env. var. idea.

Reply:
changed to use GTEST_SKIP().